db/model/Gdoc/GdocBase.ts

```diff
@@ -750,6 +751,8 @@ export class GdocBase implements OwidGdocBaseInterface {
         const chartIdsBySlug = await mapSlugsToIds(knex)
         const publishedExplorersBySlug =
             await db.getPublishedExplorersBySlug(knex)
+        const redirects = await getRedirectsFromDb(knex)
+        const resolvedRedirects = resolveRedirects(redirects)
```

Comment:
It seems we call validate on every gdoc in a loop when baking, which means we'd make the same query many times. We could move the logic to a separate function and make the query (and resolve redirects) only once.

Similarly for other DB queries we make here, but that is out of scope.

Reply:
I agree, though refactoring gdocs to validate in bulk would probably be best done in a separate PR.

As is, I've decided just to not do this validation - we've never changed an explorer to point to another explorer (except with `coronavirus-data-explorer`) so I think if it comes up again, we can just update the (likely fewer than 10) gdocs that are affected.